homeassistant/components/moehlenhoff_alpha2/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Möhlenhoff Alpha 2",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/moehlenhoff_alpha2",
-  "requirements": ["moehlenhoff-alpha2==1.2.1"],
+  "requirements": ["moehlenhoff-alpha2==1.3.0"],
```

Comment:
If you're in contact with the library maintainer, please ask them to tag github releases corresponding to the PyPI releases. That will make it easier for the community to track changes in the library.

I recommend using a gituib action to automate this.

Reply:
@j-a-n can you also create a git tag for v1.3.0?

Reply:
Just added the tag and release 👍

Reply:
Perfect, thanks!